packages/test/test-end-to-end-tests/src/test/gc/gcUnreferencedTimestamp.spec.ts

```diff
@@ -6,47 +6,39 @@
 import { strict as assert } from "assert";
 import { ContainerRuntimeFactoryWithDefaultDataStore, DataObjectFactory } from "@fluidframework/aqueduct";
 import { TelemetryNullLogger } from "@fluidframework/common-utils";
-import { IContainer } from "@fluidframework/container-definitions";
+import { IContainer, IRuntimeFactory } from "@fluidframework/container-definitions";
 import {
     ContainerRuntime,
     IAckedSummary,
     IContainerRuntimeOptions,
     SummaryCollection,
 } from "@fluidframework/container-runtime";
 import { ISummaryContext } from "@fluidframework/driver-definitions";
-import { ISummaryTree } from "@fluidframework/protocol-definitions";
+import { ISummaryTree, SummaryType } from "@fluidframework/protocol-definitions";
+import { channelsTreeName, IGarbageCollectionSummaryDetails } from "@fluidframework/runtime-definitions";
 import { requestFluidObject } from "@fluidframework/runtime-utils";
 import { ITestObjectProvider } from "@fluidframework/test-utils";
-import { describeFullCompat } from "@fluidframework/test-version-utils";
+import { describeFullCompat, describeNoCompat, getContainerRuntimeApi } from "@fluidframework/test-version-utils";
+
+import { pkgVersion } from "../../packageVersion";
 import { wrapDocumentServiceFactory } from "./gcDriverWrappers";
 import { loadSummarizer, TestDataObject, submitAndAckSummary, getGCStateFromSummary } from "./mockSummarizerClient";
 
 /**
  * Validates that the unreferenced timestamp is correctly set in the summary tree of unreferenced data stores. Also,
  * the timestamp is removed when an unreferenced data store becomes referenced again.
  */
-describeFullCompat("GC unreferenced timestamp", (getTestObjectProvider) => {
+describe("GC unreferenced timestamp", () => {
```

Comment:
Is the intention to make it full-compat in 2 versions? if so, is there a bug covering it?

Reply:
All of the current tests are still full compat in `describeFullCompat` section under this.
The new tests are a copy of one of the existing tests and it runs for different container runtime versions.